docs/advanced_usage.rst

```diff
@@ -560,6 +560,7 @@ The available environment variables for ``ddtrace-run`` are:
   any traces.
 * ``DATADOG_ENV`` (no default): Set an application's environment e.g. ``prod``,
   ``pre-prod``, ``stage``
+* ``DD_VERSION`` (no default): Set an application's version e.g. ``1.2.3``, ``6c44da20``, ``2020.02.13``
```

Comment:
should have `DD_SERVICE_VERSION` here too?

Reply:
no, `service.version` is basically the same thing as `version`, just one is our tag vs the other being OTel.